src/CentrifugeRouter.sol

```diff
@@ -3,10 +3,12 @@ pragma solidity 0.8.21;
 
 import {Auth} from "src/Auth.sol";
 import {IERC7540} from "src/interfaces/IERC7540.sol";
+import {IERC20} from "src/interfaces/IERC20.sol";
 import {SafeTransferLib} from "src/libraries/SafeTransferLib.sol";
 import {ICentrifugeRouter} from "src/interfaces/ICentrifugeRouter.sol";
+import {Multicall} from "src/Multicall.sol";
 
-contract CentrifugeRouter is Auth, ICentrifugeRouter {
+contract CentrifugeRouter is Auth, Multicall, ICentrifugeRouter {
```

Comment:
This is unrelated to this PR, but something I noticed, I wonder if we need these lines:
```
require(maxDeposit > 0, "CentrifugeRouter/user-has-no-balance-to-claim");
```

in claimDeposit/Redeem. It is anyways checked in `deposit` itself already. It is minimal but would still be a small gas saving.

Reply:
Also this was included in the spec but missed in the implementation so far, maybe you want to add this as well
```solidity
  // --- View methods ---
  function getVault(uint64 poolId, bytes16 trancheId, address asset) external view returns (address) {
    // TODO
  }
```

This will require initialising it with the pool manager.

Reply:
Updated!